README.md

```diff
@@ -10,15 +10,6 @@
   </strong>
 </p>
 
-<p align="center">
-  <a href="https://github.com/open-telemetry/opentelemetry-java-instrumentation/actions/workflows/build.yml">
-    <img alt="Build Status" src="https://img.shields.io/github/actions/workflow/status/open-telemetry/opentelemetry-java-instrumentation/build.yml?branch=main&style=for-the-badge">
-  </a>
-  <a href="https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases">
-    <img alt="GitHub release (latest by date)" src="https://img.shields.io/github/v/release/open-telemetry/opentelemetry-java-instrumentation?style=for-the-badge">
-  </a>
-</p>
-
```

Comment:
I'm not sold on the whole "TOC" header, but didn't want to remove it without thinking more.

Reply:
> I'm not sold on the whole "TOC" header, but didn't want to remove it without thinking more.

I found that [OTel Collector](https://github.com/open-telemetry/opentelemetry-collector) has also been using similar "TOC" header.😅

Reply:
went ahead and removed it, and moved the badges to the top